src/tribol/integ/FE.cpp

```diff
@@ -101,6 +101,7 @@ TRIBOL_HOST_DEVICE void SegmentBasis( const RealT* const x, const RealT pX, cons
 
   phi = 1.0 / lambda * ( lambda - magW );
 
+#ifdef TRIBOL_USE_HOST
```

Comment:
do we also want to make these slic_debug statements contingent of if defined tribol_debug? they look like debug statements.

Reply:
`SLIC_DEBUG()` macros are contingent on the def of `AXOM_DEBUG`; if we want it contingent on `TRIBOL_DEBUG`, we would need to define our own macros.  `TRIBOL_USE_HOST` is to prevent SLIC calls on device.